### PR TITLE
Issue #1565 : policy tag feature

### PR DESCRIPTION
**Pull request by sahibamittal**
Signed-off-by: Sahiba Mittal <sahibamittal98@gmail.com>

Changes for new feature to support 'Limit to Tags' in policy management.

Backend PR changes -> https://github.com/DependencyTrack/dependency-track/pull/1694

Issue: https://github.com/DependencyTrack/dependency-track/issues/1565

### CODE REVIEW

**Comment by sahibamittal** [comment]
@nscuro Could you please review again and approve the frontend PR for policy tag feature? I had to delete previous PR due to sign-off issues.

**Review by nscuro — APPROVED** [review]
LGTM. Will merge after the API server PR is merged.

**Pull request merged**
